avro/swagger/src/test/scala/com/cerner/beadledom/avro/SwaggerAvroModelConverterSpec.scala

```diff
@@ -1,55 +1,61 @@
 package com.cerner.beadledom.avro
 
+import java.io.ByteArrayOutputStream
+
 import com.cerner.beadledom.testing.JsonMatchers
 import com.google.common.base.Charsets
 import com.google.common.io.Resources
-import com.wordnik.swagger.annotations.ApiModel
-import com.wordnik.swagger.converter.ModelConverters
-import com.wordnik.swagger.core.util.JsonSerializer
+import io.swagger.converter.ModelConverters
+import io.swagger.jaxrs.listing.SwaggerSerializers
+import io.swagger.models.Swagger
+import javax.ws.rs.core.MediaType
 import org.scalatest.{BeforeAndAfterAll, FunSpec, Matchers}
 
+import scala.collection.JavaConverters
+
 /**
  * Spec tests for {@link SwaggerAvroModelConverter}.
  */
 class SwaggerAvroModelConverterSpec
-    extends FunSpec with Matchers with JsonMatchers with BeforeAndAfterAll {
+  extends FunSpec with Matchers with JsonMatchers with BeforeAndAfterAll {
   val converter = new SwaggerAvroModelConverter
 
   override def beforeAll = {
-    ModelConverters.addConverter(converter, true)
+    ModelConverters.getInstance().addConverter(converter)
   }
 
   override def afterAll = {
-    ModelConverters.removeConverter(converter)
+    ModelConverters.getInstance().removeConverter(converter)
+  }
+
+  private def serialize[T](clazz: Class[T]): String = {
+    val os = new ByteArrayOutputStream()
+
+    val models = ModelConverters.getInstance().readAll(clazz)
+
+    val swagger = new Swagger()
+    for ((key, value) <- JavaConverters.mapAsScalaMapConverter(models).asScala) {
+      swagger.addDefinition(key, value)
+    }
+
+    new SwaggerSerializers().writeTo(swagger, null, null, null, MediaType.APPLICATION_JSON_TYPE, null, os)
```

Comment:
what do all these `null`s mean?

Reply:
type, genericType, annotations, headers. None of them are actually used by the method that is being called.